### ISSUE TITLE
EventBridge PutEvents not working since 0.11.4 when Source is missing

### ISSUE THREAD

**Issue by chris922** [localstack/localstack#3043]
# Type of request: This is a ...

[X] bug report
[ ] feature request

# Detailed description

Starting with version 0.11.4 EventBridge does not work anymore.

## Expected behavior

Calling PutEvents operation towards localstack works as it is the case when using version 0.11.4.

## Actual behavior

Calling PutEvents operation towards localstack fails with the following exception:

```
botocore.exceptions.ClientError: An error occurred (500) when calling the PutEvents operation (reached max retries: 4): <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

The LocalStack container doesn't print any errors. In case I can increase verbosity please let me know.

Reproducible with versions 0.11.4 and 0.11.5, issue is not available using 0.11.3

# Steps to reproduce

1. Start LocalStack with `SERVICES=events`
2. Run the client code

## Command used to start LocalStack

```sh
docker run --rm --name localstack -p 4587:4587 -e SERVICES=events -e DEBUG=true localstack/localstack:0.11.4
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

CLI:

```
aws events put-events --endpoint-url http://localhost:4587 --entries '[{"DetailType": "Test", "Detail": "{}"}]'
```

Or via python:

```python
eb_client = boto3.client('events', endpoint_url='http://localhost:4587')
eb_client.put_events(
    Entries=[
        {
            'DetailType': 'Test',
            'Detail': '{}'
        }
    ]
)
```

**Comment by chris922** [comment]
I was able to trace it down to the missing `Source` field. Adding this field the issue is gone. Nevertheless `Source` is marked as optional according to the AWS API docs:

> **Source**
> The source of the event.
> 
> Type: String
> 
> Required: No

https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEventsRequestEntry.html

**Comment by whummer** [comment]
Thanks for reporting @chris922 . This should be fixed in #3048 - can you please pull the Docker image and give it another try with the `latest` version? Please report here if the problem persists. Thanks
